paddlecloud/paddlecloud/settings.py

```diff
@@ -277,21 +277,24 @@
 #   "dc1":{
 #       "fstype": "hostpath",
 #       "host_path": "/mnt/hdfs/",
-#       "mount_path" "/pfs/dc1/home/%s/" # mount_path % username
+#       "mount_path" "/pfs/%s/home/%s/" # mount_path % ( dc, username )
 #    }
 #}
 DATACENTERS = {
     "datacenter1":{
         "fstype": "cephfs",
-        "monitors_addr": "172.19.32.166:6789",
+        "monitors_addr": ["172.19.32.166:6789"],  # must be a list
         "secret": "ceph-secret",
         "user": "admin",
-        "mount_path": "/pfs/datacenter1/home/%s/", # mount_path % username
+        "mount_path": "/pfs/%s/home/%s/", # mount_path % ( dc, username )
         "cephfs_path": "/%s", # cephfs_path % username
         "admin_key": "/certs/admin.secret"
     }
 }
 
+# where cephfs root is mounted when using cephfs storage service
+STORAGE_PATH="/pfs"
```

Comment:
Since the "paddlecloud" API server will be deployed manually (also by k8s `yaml` file), there's no `username` for "paddlecloud" itself. And "paddlecloud" should mount the root of cephfs into the pod.